src/main/java/org/folio/rest/impl/FiscalYearAPI.java

```diff
@@ -27,6 +27,7 @@
 import io.vertx.core.Context;
 import io.vertx.core.Future;
 import io.vertx.core.Handler;
+import io.vertx.core.Promise;
```

Comment:
I understand that the `future` is deprecated now. But, since we are now using `Promise`, will there be regression failure with the rest of the project where we are still using `Future`?

Reply:
In version 3.8, the Future interface remains completable to ensure backward compatibility. https://github.com/vert-x3/issues/issues/469